shell/browser/native_window_views.cc

```diff
@@ -130,6 +125,16 @@ gfx::Rect DIPToScreenRect(HWND hwnd, const gfx::Rect& pixel_bounds) {
 
 #endif
 
+#if defined(USE_OZONE)
+
+bool CreateGlobalMenuBar() {
```

Comment:
A function named `CreateGlobalMenuBar` usually means it is used to create the menu bar, I think we should move the testing of `supports_global_application_menus` to `ShouldUseGlobalMenuBar`.

Reply:
@zcbenz, I actually [suggested he add this](https://github.com/electron/electron/pull/33355#discussion_r831707161), since it's from upstream. Agreed on the name being confusing, but that's what it's called in upstream. I was hoping to keep any new code here aligned with upstream so it'll be easier to spot divergences.

Reply:
Thanks for clarification, it makes sense then 👍 .